# gnumeric-cocoa patch 0001
# Why: Gnumeric 1.12.60 fails to build src/sstest.c with Apple clang because
#      `bool` is used without <stdbool.h>. Homebrew applies the same upstream
#      commit when building its 1.12.60 bottle. The fix is in upstream 1.12.62,
#      so drop this patch when the submodule moves past 1.12.60.
# Source: upstream commit 0de4c0a45f078ec211fd372da4103b09cb718b1b
#         (https://gitlab.gnome.org/GNOME/gnumeric/-/merge_requests/39)
From 0de4c0a45f078ec211fd372da4103b09cb718b1b Mon Sep 17 00:00:00 2001
From: botantony <antonsm21@gmail.com>
Date: Wed, 11 Feb 2026 15:07:15 +0100
Subject: [PATCH] sstest: replace `bool` type with `gboolean`

Related to https://github.com/Homebrew/homebrew-core/pull/266893

Signed-off-by: botantony <antonsm21@gmail.com>
---
 src/sstest.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/sstest.c b/src/sstest.c
index 0a0073e3e..cb86a123a 100644
--- a/src/sstest.c
+++ b/src/sstest.c
@@ -898,7 +898,7 @@ test_insert_delete (void)
 /* ------------------------------------------------------------------------- */
 
 static gboolean
-check_help_expression (const char *text, GnmFunc const *fd, bool localized)
+check_help_expression (const char *text, GnmFunc const *fd, gboolean localized)
 {
 	GnmConventions const *convs = gnm_conventions_default;
 	GnmParsePos pp;
-- 
2.50.1 (Apple Git-155)

